cmd/bbolt/command_dump.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"go.etcd.io/bbolt/internal/guts_cli"
+)
+
+func newDumpCommand() *cobra.Command {
+	var help bool
+	bucketsCmd := &cobra.Command{
+		Use:   "dump PATH pageid [pageid...]",
```

Comment:
i think having 

```
[pageid...]
```

is not user friendly, is it ?

Reply:
I think it is straightforward. Many linux tools use the same expression, for example, `man grep` describes its usage as `grep [OPTIONS] PATTERN [FILE...]`. Do you have any suggestions?